stan/math/prim/fun/read_corr_L.hpp

```diff
@@ -36,6 +36,12 @@ template <typename T>
 Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic> read_corr_L(
     const Eigen::Array<T, Eigen::Dynamic, 1>& CPCs,  // on (-1, 1)
     size_t K) {
+  if (K == 1) {
+    Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic> L(1, 1);
```

Comment:
This should be:

```
return Eigen::Matrix<T, -1, -1>::Identity(1, 1);
```

Or with `Eigen::Dynamic` instead of `-1`.

Reply:
Thanks, my code looked a bit awkward but couldn't think of a different way!